mon-pix/tests/integration/components/tutorials/card_test.js

```diff
@@ -0,0 +1,46 @@
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import { find, render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import setupIntlRenderingTest from '../../../helpers/setup-intl-rendering';
+
+describe('Integration | Component | Tutorials | Card', function () {
+  setupIntlRenderingTest();
+
+  it('renders component', async function () {
+    // given
+    this.set('tutorial', {
+      title: 'Mon super tutoriel',
+      link: 'https://exemple.net/',
+      source: 'mon-tuto',
+      format: 'vidéo',
+      duration: '00:01:00',
+      isEvaluated: true,
+      isSaved: true,
+    });
+
+    // when
+    await render(hbs`<Tutorials::Card @tutorial={{this.tutorial}} />`);
+
+    // then
+    expect(find('.tutorial-card-v2')).to.exist;
+    expect(find('.tutorial-card-v2__domain-border')).to.exist;
+    expect(find('.tutorial-card-v2__content')).to.exist;
+    expect(find('.tutorial-card-v2-content__skill'))
+      .to.have.property('textContent')
+      .that.contains("Mener une recherche et une veille d'information");
```

Comment:
```suggestion
      .that.contains("Mener une recherche et une veille d'information");
```

Questionnement: Cette assertion est validée dans ce cas puisque le contenu est en dur dans le `.hbs` mais cassera dès la mise en place du chargement de données dynamiques. Vérifier le contenu de ce `textContent` ne me paraît donc pas forcément pertinent dans ce cas.

Reply:
Ce qu'on s'est dit au moment où on l'a mis c'est que çe serait bien d'avoir ce test qui casse quand le contenu sera dynamique justement.

Reply:
Comme ça on oublie pas de modifier le test !